RecoEgamma/ElectronIdentification/plugins/ElectronRegressionValueMapProducer.cc

```diff
@@ -0,0 +1,462 @@
+                                                  sigmaIEtaIPhi,
+                                                  eMax,
+                                                  e2nd,
+                                                  eTop,
+                                                  eBottom,
+                                                  eLeft,
+                                                  eRight,
+                                                  clusterMaxDR,
+                                                  clusterMaxDRDPhi,
+                                                  clusterMaxDRDEta,
+                                                  clusterMaxDRRawEnergy,
+                                                  clusterRawEnergy0, 
+                                                  clusterRawEnergy1, 
+                                                  clusterRawEnergy2, 
+                                                  clusterDPhiToSeed0,
+                                                  clusterDPhiToSeed1,
+                                                  clusterDPhiToSeed2,
+                                                  clusterDEtaToSeed0,
+                                                  clusterDEtaToSeed1,
+                                                  clusterDEtaToSeed2,
+                                                  eleIPhi,
+                                                  eleIEta,
+                                                  eleCryPhi,
+                                                  eleCryEta);
+    } else {
+      calculateValues<EcalClusterLazyTools>(lazyTools.get(),
+                                            iEle,
+                                            iSetup,
+                                            sigmaIEtaIPhi,
+                                            eMax,
+                                            e2nd,
+                                            eTop,
+                                            eBottom,
+                                            eLeft,
+                                            eRight,
+                                            clusterMaxDR,
+                                            clusterMaxDRDPhi,
+                                            clusterMaxDRDEta,
+                                            clusterMaxDRRawEnergy,
+                                            clusterRawEnergy0, 
+                                            clusterRawEnergy1, 
+                                            clusterRawEnergy2, 
+                                            clusterDPhiToSeed0,
+                                            clusterDPhiToSeed1,
+                                            clusterDPhiToSeed2,
+                                            clusterDEtaToSeed0,
+                                            clusterDEtaToSeed1,
+                                            clusterDEtaToSeed2,
```

Comment:
oh, the unyielding desire to copy-paste.
To change this list, there are 8 places to edit.
:scream: